bin/compile

```diff
@@ -172,6 +172,8 @@ mkdir -p $BUILD_DIR/bin
 export PATH="$BUILD_DIR/bin:$PATH"
 curl --fail "${SWIFT_URL}/jq/jq" -L -s -o - > "$BUILD_DIR/bin/jq"
 chmod +x "$BUILD_DIR/bin/jq"
+curl --fail "${SWIFT_URL}/reader/reader" -L -s -o - > "$BUILD_DIR/bin/reader"
+chmod +x "$BUILD_DIR/bin/reader"
```

Comment:
reader is not such a beautiful name haha, quite generic no?

Reply:
Do you have a better one ?